Sources/Charcoal/Filters/Root/RootFilterViewController.swift

```diff
@@ -111,6 +111,12 @@ final class RootFilterViewController: FilterViewController {
         ])
     }
 
+    private func updateBottomButtonTitle() {
+        let localizedString = String(format: "showResultsButton".localized(), filter.numberOfResults)
+        let title = localizedString.replacingOccurrences(of: "\(filter.numberOfResults)", with: filter.formattedNumberOfResults)
```

Comment:
Nice 💯

Reply:
That's because `String(format:` removes whitespaces 🙈